tests/pgp.go

```diff
@@ -150,13 +149,12 @@ func init() {
 	}
 }
 
-func Sign(t *testing.T, m io.Reader) []byte {
-	t.Helper()
-	var b bytes.Buffer
-	if err := openpgp.ArmoredDetachSign(&b, keys[0], m, nil); err != nil {
-		t.Fatal(err)
+func Sign(b []byte) ([]byte, error) {
+	var buf bytes.Buffer
+	if err := openpgp.DetachSign(&buf, keys[0], bytes.NewReader(b), nil); err != nil {
```

Comment:
Any reason for this change?

Reply:
RPMs can only use Binary signatures, not armored. I don't think it mattered for the tests already written.

Reply:
Ah thanks.